### ISSUE TITLE
[Request] Could we bring in the old ritual?

### ISSUE THREAD

**Issue by iFrozenUser** [WayofTime/BloodMagic#802]
As the tittle suggest , can you guys bring back the old rituals such as

Mark Of the Falling Tower
Cry of Ethernal Soul

-= If it isn't possible , maybe you can explain the cause of it , you know , just for my knowledge (smiling face)

**Comment by iFrozenUser** [comment]
Or the flight ritual ! maybe?

**Comment by cj89898** [comment]
I really want the Cry of the Eternal Soul. I just spent the past 2 hours making a seperate T5 crafting altar thinking it was in. I will now go cry my eternal soul out :'(

**Comment by WayofTime** [comment]
These rituals will eventually be added again, but it falls under the category of "stuff from the old version may or may not be ported to the new one."

**Comment by VT-14** [comment]
I would like to say that I really like the Mark of the Falling Tower.  It's a brilliant way to gather ores without putting holes all over the world and traveling huge distances.  It's an excellent ability end-game ability to strive for.

I would make it require more resources though.  I usually skipped the Ritual of Magnetism and Ritual of the Crusher due to the Mark of the Falling Tower being not too far off (and the complexity of increasing the Ritual of Magnetism's range in 1.7.10).  When/if it is added back in, I would suggest making it cost 10 million+ LP, and make the Awakened Activation Crystal need something from the Tier 6 Altar as well.

**Comment by cj89898** [comment]
But first we need Tier 6 :D (demons)

**Comment by WayofTime** [comment]
First we need a fully fleshed out T4 and T5. T6 will be a loooooong time away.

**Comment by cj89898** [comment]
Yup! But hey I'm in for the long run with this new stuff! :D Really loving it.

**Comment by keraldi** [comment]
Could we please move stuff like this to milestones?

Mark of the Falling Tower is in the latest version.
Cry of Eternal Soul was the ritual that enables transferring LP from the network into the altar.
